tests/test_occurrences_client.py

```diff
@@ -134,6 +135,20 @@ async def test_occurrences_without_victims(occurrences_client_and_get_mock):
     )
 
 
+@mark.asyncio
+async def test_occurrences_with_format_parameter():
+    client_mock = AsyncMock()
+    metadata_mock = Mock()
+    metadata_mock.json.return_value = {"pageMeta": {"pageCount": 1}}
```

Comment:
> I decided to mock metadata (in the module it is a `dataclass`) so I can add to ir a return value.

If it is a `dataclass` (with no method `.json()`) why are you mocking a `json` method call?

If _in the module it is a `dataclass`_ you can just create an instance of this `dataclass` here ; )